dasharo-stability/capsule-update.robot

```diff
@@ -330,11 +330,13 @@ Perform Capsule Update
     Write Bare Into Terminal    reset
     Press Key N Times    1    ${ENTER}
 
-    # Confirm update by following instructions of Firmware Update Mode dialog
-    Read From Terminal Until    ${FUM_DIALOG_TOP}
-    ${out}=    Read From Terminal Until    ${FUM_DIALOG_BOTTOM}
-    ${digit}=    Get Key To Press    ${out}
-    Write Bare Into Terminal    ${digit}
+    IF    "${DMIDECODE_PRODUCT_NAME}" not in ["VP6650", "VP6670"]
+        # Confirm update by following instructions of Firmware Update Mode dialog
+        Read From Terminal Until    ${FUM_DIALOG_TOP}
+        ${out}=    Read From Terminal Until    ${FUM_DIALOG_BOTTOM}
+        ${digit}=    Get Key To Press    ${out}
+        Write Bare Into Terminal    ${digit}
+    END
```

Comment:
The dialog is gone for all platforms, so feel free to just remove it (preferably as a separate commit).